tests/java/org/pantsbuild/tools/junit/impl/ConsoleRunnerImplTest.java

```diff
@@ -1,3 +1,6 @@
+// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Since I was adding this retroactively, I wasn't sure if it should be 2015 or 2017.  I can make it 2017